pulsar-broker/src/main/java/org/apache/pulsar/common/naming/NamespaceBundles.java

```diff
@@ -204,4 +200,8 @@ public LocalPolicies toLocalPolicies() {
                 localPolicies.map(lp -> lp.getLeft().bookieAffinityGroup).orElse(null),
                 localPolicies.map(lp -> lp.getLeft().namespaceAntiAffinityGroup).orElse(null));
     }
+
+    public NamespaceName getNsname() {
+        return this.nsname;
+    }
```

Comment:
It adds a new public method to `NamespaceBundles`, which was not discussed in PIP. I think we should not add such public methods easily, especially it's a class that could be used in Pulsar plugins.

We can just move the logic to `NamespaceBundles#findBundle`:

```java
    public NamespaceBundle findBundle(TopicName topicName) {
        checkArgument(nsname.equals(topicName.getNamespaceObject()));
        return topicBundleAssignmentStrategy.findBundle(topicName, this);
    }
```